### ISSUE TITLE
google-cloud-pubsub: No such file to load -- google/gax (LoadError)

### ISSUE THREAD

**Issue by brettgriffin** [GoogleCloudPlatform/google-cloud-ruby#1562]
I'm getting an error that seems to be related to #1194 and googleapis/gax-ruby#54 despite using a version of `google-cloud-pubsub` that is greater than 0.23.0.  Can someone tell me if I'm doing something wrong here?

Error at runtime in Docker container:

```
`/` is not writable.
Bundler will use `/tmp/bundler/home/unknown' as your home directory temporarily.
/usr/local/bundle/gems/activesupport-5.0.2/lib/active_support/dependencies.rb:293:in `require': No such file to load -- google/gax (LoadError)
```

Pertinent Gemfile.lock lines:

```
google-cloud-pubsub (0.25.0)
      google-cloud-core (~> 1.0)
      google-gax (~> 0.8.0)
      grpc-google-iam-v1 (~> 0.6.8)
```

This is my Dockerfile:

```
FROM ruby:alpine

ADD Gemfile /app/
ADD Gemfile.lock /app/

RUN apk --update add --virtual build-dependencies build-base ruby-dev openssl-dev libxml2-dev libxslt-dev \
    postgresql-dev libc-dev linux-headers nodejs tzdata && \
    gem install bundler -v 1.14.6 && \
    cd /app ; bundle config build.nokogiri --use-system-libraries && bundle install --without development test

ADD . /app
RUN chown -R nobody:nogroup /app
USER nobody


WORKDIR /app

CMD ["bundle", "exec", "rails", "s", "-p", "8080"]

```

My application runs fine locally, but I suspect this is related to my Dockerfile running the application as a different user?

Thanks for any help

**Comment by blowmage** [comment]
Hi @brettgriffin, thanks for opening the issue. I've verified the file permissions in google-gax 0.8.3 are indeed missing global read. I'll escalate to the google-gax team.

**Comment by blowmage** [comment]
@brettgriffin The google-gax library just released 0.8.4 with updated file permissions. Does this new release resolve this issue for you?

**Comment by blowmage** [comment]
Closing as I believe this is resolved. Please comment if this is still an issue.
